README.md

```diff
@@ -11,19 +11,19 @@ It's part of Zenko, [Scality](http://www.scality.com/)’s
 Open Source Multi-Cloud Data Controller.
 Learn more about Zenko at [Zenko.io](http://www.zenko.io/)
 
-Backbeat is optimised for queuing metadata update and dispatching work
-to long running tasls in the background.
+Backbeat is optimized for queuing metadata updates and dispatching work
+to long-running tasks in the background.
 The core engine can be extended for many use cases,
-which are called extentions as listed below.
+which are called extensions, as listed below.
 
 ## EXTENSIONS
 
-### Asynchronous Replication
+### Cross-Region Replication
```

Comment:
I intentionally used Asynchronous here instead of Cross Region, since it's more relatable and we don't have the concept of regions.